File: spikes/m5-t1-division/testdata/broken/broken.js
// broken.js — deliberate syntax error for property P5 (fallback path).
// The unclosed template literal below leaves the brace/string scanner
// unbalanced at EOF; every strategy must still round-trip the bytes.

import fs from "fs";

export function brokenTemplate() {
  const text = `this template literal is never closed
  so the scanner hits EOF mid-string;

  function fakeInsideUnclosed() {
    return 42;
  }

File: spikes/m5-t1-division/testdata/broken/broken.py
def broken_pipeline():
    """This file contains a deliberate syntax error.

    Every strategy must still round-trip it byte-for-byte via the
    fallback path (property P5).
    """
    stages = [
        "extract",
        "transform",
        "load",
    ]
    return stages


def intentionally_broken(:
    # the colon above breaks the parse
    pass


def after_the_break():
    return "still here"
